public/controllers/register-agent/components/command-output/command-output.tsx

```diff
@@ -59,6 +95,7 @@ export default function CommandOutput(props: ICommandSectionProps) {
           )}
         </div>
         <EuiSpacer size='s' />
+        {showCommand && havePassword && <EuiSwitch checked={showPassword} label='Show password' onChange={onChangeShowPassword}/>}
```

Comment:
It is recommended that conditional rendering does not depend on &&, alternative:
`condition ? <ConditionalComponent /> : null;`

Reply:
Resolved